eng/bulkLoad/Invoke-LoadGrandBend.ps1

```diff
@@ -0,0 +1,33 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+# Runs the complete bulk upload of the Grand Bend dataset, aka "populated template"
+
+#Requires -Version 7
+$ErrorActionPreference = "Stop"
+
+Import-Module ./modules/Package-Management.psm1 -Force
+Import-Module ./modules/Get-XSD.psm1 -Force
+Import-Module ./modules/BulkLoad.psm1 -Force
+
+$baseUrl = "http://localhost:3000/local"
+$adminKey = "meadowlark_admin_key_1"
+$adminSecret = "meadowlark_admin_secret_1"
+$sampleDataVersion = "3.3.1-b"
```

Comment:
I was only testing with the "partial grand bend". I need to update the other `Invoke-Load*` scripts to match the "partial" file.